### PR TITLE
fix(curriculum): changed typo "h2" to "output"

### LINKED ISSUE #55020: Learn Recursion by Building a Decimal to Binary Converter - Step 1: Wrong element mentioned
### Describe the Issue

- In the last phrase, it is mentioned a `<h2>` element with `id="result"`, but only an `<output>` element has `id="result"`:

> "(...) and the `h2` element with the (...)."

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-recursion-by-building-a-decimal-to-binary-converter/step-1

### Your code

.

### Expected behavior

- Rectify to the following:

> "(...) and the **`output`** element with the (...)."

### Screenshots

_No response_

### System

.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by andrebdinis**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #55020

### CODE REVIEW

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/63db7f4677d06d7500a13321.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ In this project, you'll build a decimal and binary converter and learn about bot
 
 All of the HTML and CSS for this project has been provided for you.
 
-When you're ready to get started, use the `.getElementById()` method to get the input element with the id `number-input`, and store it in a variable called `numberInput`. Use the same method to get the button element with the id `convert-btn` and store it in a variable called `convertBtn`, and the `h2` element with the id `result` and store it in a variable called `result`.
+When you're ready to get started, use the `.getElementById()` method to get the input element with the id `number-input`, and store it in a variable called `numberInput`. Use the same method to get the button element with the id `convert-btn` and store it in a variable called `convertBtn`, and the `output` element with the id `result` and store it in a variable called `result`.
```
Comment (lasjorg) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/63db7f4677d06d7500a13321.md:14]:
```suggestion
When you're ready to get started, use the `.getElementById()` method to get the `input` element with the id `number-input`, and store it in a variable called `numberInput`. Use the same method to get the `button` element with the id `convert-btn` and store it in a variable called `convertBtn`, and the `output` element with the id `result` and store it in a variable called `result`.
```

**Review by lasjorg — CHANGES_REQUESTED** [review]
I think we should add backticks to all the element type for consistency.

**Review by lasjorg — APPROVED** [review]
There wasn't really a consensus on this in the issue. But I think marking the element types is probably the right call.

So I will approve this, and we can wait and see if anyone disagrees with it.

**Review by a2937 — APPROVED** [review]
In my opinion, every element type needs to be properly placed inside of backticks so that the translators know that the word actually refers to a proper HTML element as those should not get translated. 

That being said, nice find. Looks good to me. 👍

**Pull request merged**